src/Libraries/CoreNodeModelsWpf/CustomSelectionViewModel.cs

```diff
@@ -0,0 +1,48 @@
+using CoreNodeModels;
+using CoreNodeModels.Input;
+
+using Dynamo.Core;
+using Dynamo.UI.Commands;
+
+namespace CoreNodeModelsWpf
+{
+    public class CustomSelectionViewModel : NotificationObject
```

Comment:
please add /// summary tags to public members, especially when something has such a generic name such as custom selection.